src/components/NcSelect/NcSelect.vue

```diff
@@ -983,7 +983,20 @@ export default {
 	}
 }
 
+.v-select {
+	&:not(.vs--open) .vs__dropdown-toggle:hover {
+		border-color: var(--color-primary);
+	}
+
+	&.vs--open .vs__dropdown-toggle {
+		border-color: var(--color-primary);
+		border-bottom-color: transparent;
+	}
```

Comment:
Looks good!

Let's merge these declarations into the `.v-select.select` block above as well :)

Reply:
Done!